config/virtuoso/virtuoso-production.ini

```diff
@@ -0,0 +1,249 @@
+;
+;  virtuoso.ini
+;
+;  Configuration file for the OpenLink Virtuoso VDBMS Server
+;
+;  To learn more about this product, or any other product in our
+;  portfolio, please check out our web site at:
+;
+;      http://virtuoso.openlinksw.com/
+;
+;  or contact us at:
+;
+;      general.information@openlinksw.com
+;
+;  If you have any technical questions, please contact our support
+;  staff at:
+;
+;      technical.support@openlinksw.com
+;
+;
+;  Database setup
+;
+[Database]
+DatabaseFile       = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.db
+ErrorLogFile       = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.log
+LockFile           = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.lck
+TransactionFile    = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.trx
+xa_persistent_file = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.pxa
+ErrorLogLevel      = 7
+FileExtend         = 200
+MaxCheckpointRemap = 20000
+Striping           = 0
+TempStorage        = TempDatabase
+Syslog             = 0	; write to syslog in case of failures where writing to log is not possible
+
+[TempDatabase]
+DatabaseFile       = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso-temp.db
+TransactionFile    = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso-temp.trx
+MaxCheckpointRemap = 000
```

Comment:
It's not clear to me why this is 0 or what that means in practice. In QA it's 2000.

Reply:
I'm not sure either. I left the production config (taken from the prod server) unchanged, as I don't really understand all the parameters.